src/sqlancer/stonedb/gen/StoneDBExpressionGenerator.java

```diff
@@ -300,7 +300,7 @@ public String getTextRepresentation() {
      * Bitwise operators supported by StoneDB: https://stonedb.io/docs/SQL-reference/operators/bitwise-operators
      */
     public enum StoneDBBinaryBitwiseOperator implements Operator {
-        AND("&"), OR("|"), XOR("^"), LEFTSHIFT("<<"), RIGHTSHIFT(">>");
```

Comment:
XOR is supported, though? For unfixed bugs, we should adopt the pattern at https://github.com/sqlancer/sqlancer/blob/main/CONTRIBUTING.md#unfixed-bugs.

Reply:
sorry for the confusion. I made a mistake in the impelmentation. `XOR` is not `StoneDBBinaryBitwiseOperator` so I delete it here. `XOR` should be `StoneDBBinaryLogicalOperator` and it already exists so it is not shown in the diff.

<img width="939" alt="image" src="https://github.com/sqlancer/sqlancer/assets/63448884/87195172-fece-49b0-b238-c6500bf33277">
<img width="461" alt="image" src="https://github.com/sqlancer/sqlancer/assets/63448884/e3e2a0aa-1df7-46e9-bd79-0d4d6aa4018d">
<img width="628" alt="image" src="https://github.com/sqlancer/sqlancer/assets/63448884/a5020de5-b397-4547-91b1-fe235840a3c8">

Reply:
Ohh, Sure, I have made a mistask.